docs/source/search-syntax.rst

```diff
@@ -72,13 +73,47 @@ For example:
   metrics."2019-04-02 error rate"
 
 
+Run attributes
+~~~~~~~~~~~~~~
+
+Syntax supports searching runs using two attribute names ``status`` and ``artifact_uri``. Search
+will error out if other attribute names used in filter string. Both these attributes have string
+values.
+
+Experiment ID field is already a part of Search API. Run's ``lifecycle_stage`` attribute is not
+allowed in search because it is already encoded as a part of API's ``run_view_type`` field. To
+search for runs using ``run_id`` it is more efficient to use get run APIs. Other fields in
+:py:class:`mlflow.entities.RunInfo` are `System tags`_.
+
+Currently, search does not support ``start_time`` and ``end_time`` attributes, since values have
```

Comment:
It's a little odd to give a technical reason for why these are not supported. If we want to call them out explicitly, maybe we could just say, "Search does not currently support ``start_time`` and ``end_time`` attributes, but these will be added in a future version."

Reply:
sounds good.